myuw/templates/handlebars/card/seattle_application.html

```diff
@@ -40,14 +32,13 @@ <h4 class="myuw-card-heading myuw-header-msg"> For application status, contact t
                     <div class="myuw-card-row-content">225 Schmitz Hall</div>
                 </div>
             </div>
-            <div class="row last">
-                <div class="col-sm-3">
-                    <div class="myuw-card-row-heading">Mail</div>
-                </div>
-                <div class="col-sm-9">
-                    <div class="myuw-card-row-content">Registration and Transcripts Office<br>University of Washington <br>Box 355850 <br>Seattle, WA 98195-5850</div>
-                </div>
-            </div>
+	</div>
+	<div class="myuw-card-section" style="padding-top: 8px;">
```

Comment:
Not a big deal, but you don't really need to add a "myuw-card-section" class to normal content like this (IMO). Those were meant to be wrapper classes for the two column layout content to create sections with horizontal lines between them.